### PR TITLE
Actually hide hidden item actions

### PR DESCRIPTION
**Pull request by zepfietje**
## Description

Repeater and builder item actions that are conditionally hidden/visible still render as disabled. This PR fixes that bug by checking if the action is visible for the current item before rendering the action.

## Functional changes

- [x] Code style has been fixed by running the `composer cs` command.
- [x] Changes have been tested to not break existing functionality.
- [x] Documentation is up-to-date.

### CODE REVIEW

File: packages/forms/resources/views/components/repeater/index.blade.php [hunk 1]
```diff
@@ -79,7 +82,16 @@ class="items-start gap-4"
                                 $extraItemActions,
                                 fn (Action $action): bool => $action(['item' => $uuid])->isVisible(),
                             );
-                            $itemHasToolbar = $isReorderableWithDragAndDrop || $isReorderableWithButtons || filled($itemLabel) || $isCloneable || $isDeletable || $isCollapsible || $visibleExtraItemActions;
+                            $cloneAction = $cloneAction(['item' => $uuid]);
```
Comment (danharrin) [inline packages/forms/resources/views/components/repeater/index.blade.php:85]: I don't think you need to reassign the var, invoking it modifies the existing object

**Review by danharrin — CHANGES_REQUESTED** [review]
Please do the same for the builder & simple repeater

File: packages/forms/resources/views/components/repeater/index.blade.php (hunk 1 above)
Comment (zepfietje) [reply packages/forms/resources/views/components/repeater/index.blade.php:85]: I just tried and it doesn't, so will need to stay as is.

**Comment by zepfietje** [comment]
Done, @danharrin! :)

**Pull request merged**